src/wasmv1_execution/abi/abis.rs

```diff
@@ -931,7 +1055,8 @@ fn abi_caller_has_write_access(
         store_env,
         arg_offset,
         |handler, _req: CallerHasWriteAccessRequest| -> Result<AbiResponse, WasmV1Error> {
-            match handler.interface.caller_has_write_access() {
+            let interface = handler.exec_env.get_interface();
```

Comment:
can be inline, even better make interface pub

Reply:
See previous comment